client/data/deposits/actions.js

```diff
@@ -1,9 +1,21 @@
 /** @format */
 
+/**
+ * External dependencies
+ */
+import { apiFetch, dispatch } from '@wordpress/data-controls';
```

Comment:
Any know reason for not using `dispatch` from `@wordpress/data`? [seems to be deprecated now].(https://github.com/WordPress/gutenberg/blob/trunk/packages/data-controls/CHANGELOG.md#deprecations)

Reply:
Known reason, no. I tried swapping it out, and it appears the syntax itself is different with how it's called, so it's not a simple swap out. Since dispatch is used in the same across the client, it may be best to follow that for now and open a new issue to do a refactor to update all of the calls at the same time. 

The new dispatch function is here:
https://github.com/WordPress/gutenberg/blob/v10.4.0/packages/data/src/registry.js#L128-L146

I've created an issue here:
https://github.com/Automattic/woocommerce-payments/issues/1574

I also noticed that `apiFetch` is available in `@wordpress/api-fetch`, so I created an issue about that, as well.
https://github.com/Automattic/woocommerce-payments/issues/1575

Reply:
💯 Agree! and thanks for creating those two issues.
Good catch at the `apiFetch`, I used it from `@wordpress/api-fetch` during my trial.